### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by tarunsankhla**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,58 +1,70 @@
...
-*{
-  padding: 0;
-  margin: 0;
-  box-sizing: border-box;
+body {
+	/* font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+		"Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+		sans-serif;
+	-webkit-font-smoothing: antialiased; */
+	/* -moz-osx-font-smoothing: grayscale; */
```
Comment (pankaj-prog) [inline src/App.css]: remove commented code if no longer useful

File: src/Main.tsx [hunk 2]
```diff
@@ -1,25 +1,33 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { Outlet } from 'react-router'
 import { VAR_ENCODE_TOKEN, VAR_USER_ID } from './utils/Route';
 import { ToastContainer, toast } from 'react-toastify';
 import 'react-toastify/dist/ReactToastify.css';
 import Navbar from './components/common/Navbar/Navbar';
 import Footer from './components/common/Footer/Footer';
 import AsideNavbar from './components/common/AsideNavbar/AsideNavbar';
```
Comment (pankaj-prog) [inline src/Main.tsx:7]: suggestion: you can import components from a single file, that will look much cleaner and readable

File: src/components/UI/Modal/FullPageModal/FullPageModal.css [hunk 3]
```diff
@@ -0,0 +1,27 @@
+.modal-fixed-bg-highlight{
+    position: fixed;
+    height: 100vh;
+    width: 100vw;
+    background-color: rgba(0, 0, 0, 0.31);
```
Comment (pankaj-prog) [inline src/components/UI/Modal/FullPageModal/FullPageModal.css:5]: use color variable

File: src/pages/LoginPage/LoginPage.css [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+	border-radius: var(--border-radius);
+	font-size: 1.5rem;
+	padding: 2rem 0;
+	gap: 1em;
+	border-radius: 2em;
+	/* box-shadow: 0 4px 2px rgba(0,0,0,0.02),
+    0 8px 16px rgba(0,0,0,0.03),
+    0 16px 10px rgba(0,0,0,0.04),
+    0 26px 18px rgba(0,0,0,0.05); */
+}
```
Comment (pankaj-prog) [inline src/pages/LoginPage/LoginPage.css]: remove commented code

File: src/pages/LoginPage/LoginPage.tsx [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+                            Remember me
+                        </div>
+                        <div className="btn-link">Forgot your password?</div>
+                    </div>
+                    <div className="login-btn-container">
+                        <div className="btn login-action-btn"
+                            onClick={onSubmitHandler}>Login</div>
+                    </div>
+                    <div className="login-btn-container">
+                        <div className="btn login-action-btn"
```
Comment (pankaj-prog) [inline src/pages/LoginPage/LoginPage.tsx]: - you should use button here
- in this file, you have used div mostly, use semantic HTML

**Review by pankaj-prog — APPROVED** [review]

File: src/App.tsx [hunk 6]
```diff
@@ -19,27 +20,34 @@ function App() {
...
         <Route element={<Main />}>
           {/* <Route path='/' element={<HomePage />}></Route> */}
           <Route path={ROUTE_PATH_LandingPage} element={
-                <Suspense fallback={<h1>Loading Home...</h1>}>
-                  <HomePage />
-                </Suspense>} />
+            <Suspense fallback={<h1>Loading Home...</h1>}>
+              <HomePage />
+            </Suspense>}
+          />
```
Comment (m26unkwn) [inline src/App.tsx:31]: like that you are using code splitting for the perfomance

File: src/pages/SignUpPage/SignUpPage.tsx [hunk 7]
```diff
@@ -0,0 +1,217 @@
...
+      //         var user = res?.data?.createdUser;
+      //         var userId =res?.data?.createdUser._id;
+      //         localStorage.setItem(VAR_USER_ID, userId);
+      //         userDispatch({ email: res.data.createdUser.email, firstName: res.data.createdUser.firstName, lastName: res.data.createdUser.lastName })
+      //         setlogin(true);
+      //         Alert("success", "SuccessFully Logged In!!");
+      //         navigate("/");  
+      //     }
+
+    }
```
Comment (m26unkwn) [inline src/pages/SignUpPage/SignUpPage.tsx]: remove unwanted comments

**Review by m26unkwn — APPROVED** [review]
Like that you are using typescript 
You can look for the comments given by me and others
than it is good to merge

File: src/App.css (hunk 1 above)
Comment (tarunsankhla) [reply src/App.css]: done

File: src/Main.tsx (hunk 2 above)
Comment (tarunsankhla) [reply src/Main.tsx:7]: i thought of it but they are not that frequently imported only here

File: src/pages/LoginPage/LoginPage.css (hunk 4 above)
Comment (tarunsankhla) [reply src/pages/LoginPage/LoginPage.css]: done

File: src/pages/LoginPage/LoginPage.tsx (hunk 5 above)
Comment (tarunsankhla) [reply src/pages/LoginPage/LoginPage.tsx]: okay

File: src/pages/SignUpPage/SignUpPage.tsx (hunk 7 above)
Comment (tarunsankhla) [reply src/pages/SignUpPage/SignUpPage.tsx]: done

**Pull request merged**